package.json

```diff
@@ -32,10 +33,11 @@
     "typesense-instantsearch-adapter": "^2.4.0"
   },
   "peerDependencies": {
+    "@babel/runtime": "^7.17.9",
```

Comment:
`@babel/runtime` should also be set as an optional peer dependency because projects could have some other transpiration mechanism built-in, like in the case of docusaurus.